scipy/stats/_morestats.py

```diff
@@ -248,44 +248,50 @@ def kstat(data, n=2, *, axis=None):
 
     Notes
     -----
-    For a sample size n, the first few k-statistics are given by:
+    For a sample size :math:`n`, the first few k-statistics are given by
 
     .. math::
 
-        k_{1} = \mu
-        k_{2} = \frac{n}{n-1} m_{2}
-        k_{3} = \frac{ n^{2} } {(n-1) (n-2)} m_{3}
-        k_{4} = \frac{ n^{2} [(n + 1)m_{4} - 3(n - 1) m^2_{2}]} {(n-1) (n-2) (n-3)}
+        k_1 &= \frac{S_1}{n}, \\
+        k_2 &= \frac{nS_2 - S_1^2}{n(n-1)}, \\
+        k_3 &= \frac{2S_1^3 - 3nS_1S_2 + n^2S_3}{n(n-1)(n-2)}, \\
+        k_4 &= \frac{-6S_1^4 + 12nS_1^2S_2 - 3n(n-1)S_2^2 - 4n(n+1)S_1S_3
+        + n^2(n+1)S_4}{n (n-1)(n-2)(n-3)},
 
-    where :math:`\mu` is the sample mean, :math:`m_2` is the sample
-    variance, and :math:`m_i` is the i-th sample central moment.
+    where
+
+    .. math::
+
+        S_r \equiv \sum_{i=1}^n X_i^r,
+        
+    and :math:`X_i` is the :math:`i` th data point.
 
     References
     ----------
-    http://mathworld.wolfram.com/k-Statistic.html
+    .. [1] http://mathworld.wolfram.com/k-Statistic.html
 
-    http://mathworld.wolfram.com/Cumulant.html
+    .. [2] http://mathworld.wolfram.com/Cumulant.html
 
     Examples
     --------
     >>> from scipy import stats
     >>> from numpy.random import default_rng
     >>> rng = default_rng()
 
-    As sample size increases, n-th moment and n-th k-statistic converge to the
+    As sample size increases, `n`-th moment and `n`-th k-statistic converge to the
     same number (although they aren't identical). In the case of the normal
     distribution, they converge to zero.
 
-    >>> for n in [2, 3, 4, 5, 6, 7]:
-    ...     x = rng.normal(size=10**n)
+    >>> for i in range(2,8):
```

Comment:
The use of `n` here to change the size seemed confusing because `n` also denoted the nth moment